app/controllers/users/nbp_wallet_controller.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
+
+module Users
+  class NbpWalletController < ApplicationController
+    CONNECTOR_URL = 'http://localhost:80'
+    API_KEY = 'myapikey'
+    DISPLAY_NAME = 'CodeHarbor'
```

Comment:
At least those are a good candidate for Settings (as you probably intended anyway).